bentoml/yatai/db.py

```diff
@@ -46,11 +46,9 @@ def init_db(db_url):
     engine = create_engine(db_url, **extra_db_args)
 
     if not database_exists(engine.url) and not is_sqlite_db(db_url):
-        raise BentoMLException(
-            f'Database does not exist or Database name is missing in config '
-            f'db.url: {db_url}'
-        )
-    create_all_or_upgrade_db(engine, db_url)
+        logger.debug(f'Creating BentoML database at {engine.url}')
+        create_database(engine.url)
```

Comment:
add a test for this?  

Does this work with the local sqlite situation? Could you verify the e2e yatai_service tests all pass with this change?

Reply:
This line is not related to the issue in https://github.com/bentoml/BentoML/discussions/1053 right? What's the reason or adding it?

Reply:
I was thinking to make people's life easier since they don't have to create the database themselves.  I will remove this and start a separate PR for it.